content/300-guides/200-deployment/301-edge/index.mdx

```diff
@@ -11,6 +11,6 @@ If your application is deployed via a Function-as-a-Service (FaaS) offering and
 
 </TopBlock>
 
-## Edge (Distributed FaaS) guides
+## Guides for Distributed Function-as-a-Service providers
```

Comment:
I might have potentially said something other in the past, but this should probably say "Edge Functions" somehow.